bBridgeAPISDK/Common/Authorization/CredentialsAuthorizer.cs

```diff
@@ -23,7 +22,7 @@ public class LazyCredentialsAuthorizer: IAuthorizer
         /// </summary> 
         /// <param name="username">Name of bBridge API user</param>
         /// <param name="password">Password of bBridge API user</param>
-        public LazyCredentialsAuthorizer(string username, string password, Uri authBaseUri)
+        public LazyCredentialsAuthorizer(string username, string password, string authBaseUri)
```

Comment:
Maybe pass `IAsyncHttpRequester` as constructor param instead of manual creation?
Now `HttpRequester` contains `IAuthorizer` and if `IAuthorizer` is `LazyCredentialsAuthorizer` it has own `HttpRequester` - looks very complicated